ipa-server/requirements.yml

```diff
@@ -0,0 +1,8 @@
+# This is the Ansible Galaxy requirements file to pull in the correct roles
+# to support the operation of CASL provisioning/runs.
+
+# From 'casl-ansible'
+- name: openshift-applier
+  scm: git
+  src: https://github.com/redhat-cop/openshift-applier
+  version: v2.0.10
```

Comment:
Please update to v2.1.1

Reply:
@oybed I'm actually seeing some issues when changing from v2.0.10 to v.2.1.1. Probably something small, but not sure I'll have time to resolve before I leave. Could we look at leaving at the current version and then resolving in a follow-up PR?

Reply:
sounds good